src/h5web/metadata-viewer/EntityTable.tsx

```diff
@@ -31,8 +31,8 @@ function EntityTable(props: Props) {
           <td>{path}</td>
         </tr>
         <tr>
-          <th scope="row">Name</th>
-          <td>{path === '/' ? filename : name}</td>
+          <th scope="row">{path === '/' ? 'File path' : 'Name'}</th>
+          <td>{path === '/' ? filepath : name}</td>
```

Comment:
Maybe introduce an `isRoot` or `isRootEntity` constant for clarity?

Reply:
Agreed, I considered doing that :+1: